src/security/builtin_plugins/cryptographic/src/crypto_transform.c

```diff
@@ -1192,9 +1194,8 @@ encode_datawriter_submessage_encrypt (
     }
   }
   else
-  {
```

Comment:
I wouldn't remark on this if I had no other comments, but: as rule I don't like ``if(P) X else Y`` where X is significantly longer than Y, and I really don't like Y to then be a single statement without braces. Just so you know :)